README.md

```diff
@@ -283,6 +283,14 @@ Commands:
 
   patch <ID> <FILE>          Apply patch FILE to ID.
 
+  env-get                    Get the complete environment of the cluster master
```

Comment:
I can add `env[-get]` as well as allowing it to accept a target. For the filter aspect, I think sorting it is close enough to filtering since at this level the user could be expected to pipe the output through grep.